pyproject.toml

```diff
@@ -8,7 +8,7 @@ version = "0.1.0"
 authors = [
     { name = "Canonical Public Cloud", email = "cpc@canonical.com" }
 ]
-description = "Basic Azure Image Testing for Linux API client"
+description = "Azure Image Testing for Linux API (AITL) client"
```

Comment:
The `AITL` acronym does not include `API` and thus should precede `API` (i.e., `...(AITL) API`).

Reply:
This still needs to be addressed.